### PR TITLE
Remove support for GopherJS

### PR DESCRIPTION
**Pull request by Jacalz**
WIP PR for removing the support for using GopherJS.

### CODE REVIEW

File: cmd/fyne/internal/commands/package-wasm.go [hunk 1]
```diff
@@ -8,22 +8,6 @@ import (
 	"fyne.io/tools/cmd/fyne/internal/templates"
 )
 
-func (p *Packager) packageWeb() error {
```
Comment (andydotxyz) [inline cmd/fyne/internal/commands/package-wasm.go:11]: Shouldn't this wasm-only package format be called "web"?
The fact that it's a wasm binary internally should be considered an implementation detail I think.
So let's use the target name web, and use that for the folder as well.

Not sure we have to change this function name, but the file rename and folder here seemed like a good place to comment

**Review by andydotxyz — CHANGES_REQUESTED** [review]
Thanks for this, but can't this target be called "web" not "wasm"?
That way users won't have to change their behaviour from 2.4 unless they were previously specifying a subset of the web package?

**Comment by Jacalz** [comment]
I was going to say that all other targets matched an operating system and that we ought to stick to wasm for consistency, but then I checked and wasm is the architecture and not the os. Technically we should do `js`, I guess, but maybe we can make `web` be an alias for that?

**Comment by Jacalz** [comment]
(So yes, I agree but maybe we ought to use `js` instead of `web` as the os target)

**Comment by andydotxyz** [comment]
I think it would be very confusing if we move to "js" and yet are removing the GopherJS support.
Also JS is a language and could imply that we are transpiling which is not what the web/WASM is doing.

I guess I don't see why we should move from the existing os=web which has been in place for over a year...

**Comment by Jacalz** [comment]
Yeah, you are right. Using js would be more consistent with Go tooling but yeah (Go wasm builds actually have GOOS=js), it does give the wrong impression. I'll keep web and wasm around.

**Comment by Jacalz** [comment]
I think it should be possible to merge this and pin the repo to the organization after that :)

**Comment by andydotxyz** [comment]
Is it intended to keep the "js" os value present?

**Comment by Jacalz** [comment]
Hmm. I forgot to remove hit but it is the actual GOOS for wasm so maybe we want to support it?

**Review by andydotxyz — APPROVED** [review]
OK, thanks - it seems like the right thing to do now.
We can always review if GopherJS gets a whole new lease of life

**Comment by Jacalz** [comment]
You can pin the repo now :)

**Comment by andydotxyz** [comment]
Done :)

**Pull request merged**